docs-conceptual/azurermps-5.7.0/install-azurerm-ps.md

```diff
@@ -38,161 +40,137 @@ PowerShellGet 1.6.0   C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1
 PowerShellGet 1.0.0.1 C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PowerShellGet.psd1
 ```
 
-You need PowerShellGet version 1.1.2.0 or higher. To update PowerShellGet, use the following command:
+If you need to update your installation of PowerShellGet, run the following command:
 
 ```powershell
 Install-Module PowerShellGet -Force
 ```
 
-If you do not have PowerShellGet installed, see the [How to get PowerShellGet](#how-to-get-powershellget)
-section of this article.
+If you don't have PowerShellGet installed, follow the instructions in the table below for your system.
+
+|Scenario|Install instructions|
+|---|---|
+|Windows 10<br/>Windows Server 2016|Built into Windows Management Framework (WMF) 5.0 included in the OS|
+|Upgrade to PowerShell 5|<ol><li>[Install the latest version of WMF](https://www.microsoft.com/en-us/download/details.aspx?id=54616)</li><li>Run the following command:<br/>```Install-Module PowerShellGet -Force```</li></ol>|
+|Windows with PowerShell 3 or PowerShell 4|<ol><il>[Get the PackageManagement modules](http://go.microsoft.com/fwlink/?LinkID=746217)</il><li>Run the following command:<br/>```Install-Module PowerShellGet -Force```</li></ol>|
 
 > [!NOTE]
 > Using PowerShellGet requires an Execution Policy that allows you to run scripts. For more
 > information about PowerShell's Execution Policy, see
 > [About Execution Policies](/powershell/module/microsoft.powershell.core/about/about_execution_policies).
 
-## Step 2: Install Azure PowerShell
+## Install the Azure PowerShell module
 
 Installing Azure PowerShell from the PowerShell Gallery requires elevated privileges. Run the
 following command from an elevated PowerShell session:
 
 ```powershell
-# Install the Azure Resource Manager modules from the PowerShell Gallery
-Install-Module -Name AzureRM -AllowClobber
+Install-Module -Name AzureRM 
```

Comment:
If you are removing the information about the AllowClobber (below) then you need to add -AllowClobber to this command. That will avoid the error and the need for documenting it.